docker/viame_girder_worker_gpu.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM kitware/viame:girder-worker-gpu-latest
+
+USER root
+
+WORKDIR /home
+
+RUN cd /viame/cmake \
+	&& chmod +x build_server_ubuntu.sh \
+        && sleep 1 \
+ 	&& ./build_server_ubuntu.sh
```

Comment:
I don't see why this is being done in this dockerfile rather than in the base image.  It has no dependencies.